tg_bot/__main__.py

```diff
@@ -387,7 +386,7 @@ def CoCDone(bot: Bot, update: Update):
         )
     else:
         update.effective_message.reply_text(
-            "Non hai letto le CoC? Sei una delusione... \nNon sei abilitato per questo comando.",
+            "Non hai letto le CoC? Prima di interagire nel gruppo devi prendere visione e accettare il Codice di Condotta... \nNon sei abilitato per questo comando.",
```

Comment:
"Le CoC" va bene? Diciamo "il Codice di Condotta" al maschile nella stessa frase. Magari ridurrei il virgolettato a soltanto la parte centrale:
```suggestion
            "Prima di interagire nel gruppo devi prendere visione e accettare il Codice di Condotta.",
```